docs/source/api/apollo-server.mdx

```diff
@@ -265,37 +240,33 @@ To learn more about configuring Apollo Server's cache, see [Configuring cache ba
 
```

Comment:
I would probably rename "Networking options" to "Transport options" or "Protocol options", and move persistedQueries into it.

Reply:
I like `Protocol options`! Did this and moved `persistedQueries` in.